ocaml/xapi/xapi_pool_update.ml

```diff
@@ -240,6 +241,9 @@ let parse_update_info xml =
     let name_label = try List.assoc "name-label" attr
       with _ -> raise (Api_errors.Server_error(Api_errors.invalid_update, ["missing <name-label> in update.xml"]))
     in
+    let version = try List.assoc "version" attr
```

Comment:
Small suggestions: (1) a more conventional formatting of `try-with` and (2) catching specifically `Not_found` to avoid catching unrelated exceptions (which would not happen here).

```
let version = try List.assoc "version" attr with
 Not_found -> raise (Api_errors.Server_error(Api_errors.invalid_update, ["missing <name-label> in update.xml"]))
```